### ISSUE TITLE
URLs in markdown are not auto linked

### ISSUE THREAD

**Issue by chinglinwen** [go-gitea/gitea#3446]
- Gitea version (or commit ref): Gitea Version: 1.3.2
- Git version: git version 1.8.3.1 (server)
- Operating system: ( centos7)
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL) https://try.gitea.io/33485938475983475/DeezLoader-Reborn/issues/2
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

in issue comments or markdown editor

enter a link 

```
http://example.com
```
expect it auto linked

expect to no need to enter the following

```
[http://example.com](http://example.com)
```

**Comment by captn3m0** [comment]
Auto-linking is a feature of GitHub formatted markdown, and not standard markdown. You should be using `<http://example.com>` instead.

**Comment by chinglinwen** [comment]
That way is a lot easier than the default one, it helps a lot for me, thanks.  though, I think the Auto-linking  feature should still be added.

**Comment by thehowl** [comment]
Solved by #3354 (merged)
